### DISCUSSION TITLE
Using custom utilities inside custom utilities

### DISCUSSION THREAD

**Discussion by simonhammerholt** [tailwindlabs/tailwindcss#4061] [Help] [answered]
Hi everyone. 

We've started using tailwind, and with it, we've started adding custom utilities for often used utility combinations, such as font-size, color and weight for different sections in both light and dark mode.

In relation to that, we've found that it would be beneficial, to use these custom utilities in other custom utilities, such as utilities that define specific areas including the above-mentioned utilities as well as background-color, border-color padding etc.

Is this possible? And in case it is, how would you do this?

**Comment by olets** [comment, marked as answer]
You can [`apply`](https://tailwindcss.com/docs/functions-and-directives#apply) custom utilities and components in your custom utilities and components.

A minimal example of `apply`ing a custom utility:

```css
@layer utilities {
  .my-utility {
    @apply text-red-400;
  }

  .my-other-utility {
    @apply my-utility;
  }
}
```

A minimal example of `apply`ing utilities defined in a stylesheet or in the config: https://play.tailwindcss.com/kW7DMucX1N

**Reply by simonhammerholt** [reply]
Perfect. For some reason i couldn't get that working, but it works now. Thank you!
